### DISCUSSION TITLE
Saving scene

### DISCUSSION THREAD

**Discussion by RhythmOfTension** [pelicanmapping/osgearth#1837] [Support Q&A] [answered]
Hi, 

Is there a way to save region of osgEarth scene with imagelayers, elevationlayers, annotations, featuremodellayers (or some of them) to one of well known 3d formats?

If there are none currently from API, is it possible by design? Or I just need to forget about this idea?

Thanks.

**Comment by gwaldron** [comment, marked as answer]
No, this is not supported. Forget about it! :)
